src/kv/store.h

```diff
@@ -126,6 +129,17 @@ namespace kv
       history = history_;
     }
 
+    std::shared_ptr<ccf::ProgressTracker> get_progress_tracker()
```

Comment:
If the store doesn't use `progress_tracker`, why is it set on the store and not on `AFT` directly?

Reply:
Store is where we store history & consensus and this class is meant to have the same scope. Importantly, store is where other classes get access to the history and consensus classes